en/app.ftl

```diff
@@ -310,6 +310,17 @@ faq-question-8-answer-html = You can learn more about the data { -brand-name-fir
 faq-question-email-storage-question = Does { -brand-name-relay } store my emails?
 faq-question-email-storage-answer = Under the rare circumstance in which the service is down, we may temporarily store your emails until we are able to send them. We will never store your emails for longer than three days.
 
+faq-question-acceptable-use-question = What are the acceptable uses of Relay?
```

Comment:
Should Relay in this string be `-brand-name-relay`?

Reply:
Oof yes, good eye, thanks. Fixed both this and the Mozilla brand name variable.